uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/LeaveBalanceChange.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/HRLeaveAndAbsences</base>
+  <class>LeaveBalanceChange</class>
+  <name>civicrm_hrleaveandabsences_leave_balance_change</name>
+  <comment>Store balance changes to a Leave Period Entitlement</comment>
+  <add>4.4</add>
+  <log>true</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>Unique LeaveBalanceChange ID</comment>
+    <add>4.4</add>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <name>entitlement_id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>FK to LeavePeriodEntitlement</comment>
+    <add>4.4</add>
+  </field>
+  <foreignKey>
+    <name>entitlement_id</name>
+    <table>civicrm_hrleaveandabsences_leave_period_entitlement</table>
+    <key>id</key>
+    <add>4.4</add>
+    <onDelete>CASCADE</onDelete>
+  </foreignKey>
+
+  <field>
+    <name>type_id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>One of the values of the Leave Balance Type option group</comment>
+    <add>4.4</add>
+    <pseudoconstant>
+      <optionGroupName>hrleaveandabsences_leave_balance_change_type</optionGroupName>
+    </pseudoconstant>
+  </field>
+
+  <field>
+    <name>amount</name>
+    <type>decimal</type>
```

Comment:
on the ticket description :

`amount: type=decimal(5,2)`

but here in the xml file you don't define the the precision (5) nor the scale ( 2) , I don't know if that even possible to do in the xml file .